### PR TITLE
[ois] Update to 1.5.1

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Updates ois to 1.5.1

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  As before, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run ./vcpkg x-add-version --all and committed the result?  
  Yes

### CODE REVIEW

File: ports/ois/portfile.cmake [hunk 1]
```diff
@@ -5,27 +5,26 @@ vcpkg_fail_port_install(ON_ARCH "arm" "arm64" ON_TARGET "uwp")
 vcpkg_from_github(
```
Comment (NancyLi1013) [inline ports/ois/portfile.cmake:5]: The above message can be removed now. You can add `supports: ! (arm | uwp)` in the vcpkg.json if the new version still doesn't support arm and uwp.

Besides this, please also remove the failures in ci.baseline.txt.
https://github.com/microsoft/vcpkg/blob/66c39e113a348a6ae80419fae5a629b951b00f1a/scripts/ci.baseline.txt#L1032-L1034

File: ports/ois/portfile.cmake [hunk 2]
```diff
@@ -5,27 +5,26 @@ vcpkg_fail_port_install(ON_ARCH "arm" "arm64" ON_TARGET "uwp")
...
 
 # Include files should not be duplicated into the /debug/include directory
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/LICENSE.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
-
-vcpkg_fixup_pkgconfig()
+file(INSTALL "${SOURCE_PATH}/LICENSE.md" DESTINATION" ${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```
Comment (NancyLi1013) [inline ports/ois/portfile.cmake:30]:
```suggestion
file(INSTALL "${SOURCE_PATH}/LICENSE.md" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/ois/portfile.cmake (hunk 1 above)
Comment (chausner) [reply ports/ois/portfile.cmake:1]: I assume you meant `supports: ! (arm | arm64 | uwp)`? I made the corresponding changes in https://github.com/microsoft/vcpkg/pull/21244/commits/781a71e2db84eecbc8e4d630d7a701e63d860a1a.

Comment (NancyLi1013) [reply ports/ois/portfile.cmake:1]: Actually, only arm is enough. Of course, you can also add arm64.

**Review by NancyLi1013 — APPROVED** [review]

**Pull request merged**